environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -878,6 +898,220 @@ objects:
         - name: telemeter-server-tls
           secret:
             secretName: telemeter-server-shared
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  rules:
+  - nonResourceURLs:
+    - /metrics
+    verbs:
+    - get
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: prometheus-ams-${NAMESPACE}
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-ams
+    namespace: ${NAMESPACE}
+- apiVersion: v1
+  data:
+    nginx.conf: |
+      daemon off;
+      worker_processes 1;
+      error_log /dev/stderr;
+      pid /tmp/nginx.pid;
+
+      events {
+          worker_connections 1024;
+      }
+
+      http {
+        log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                          '$status $body_bytes_sent "$http_referer" '
+                          '"$http_user_agent" "$http_x_forwarded_for"';
+
+        upstream app {
+          server http://thanos-receive.${NAMESPACE}.svc.cluster.local:10902/api/v1/receive;
```

Comment:
I anticipated OpenShift templating engine would change this as well.

Reply:
That is correct.